playbooks/site.yml

```diff
@@ -61,3 +228,85 @@
       with_items:
       - "{{ users + password_users }}"
       when: item.state == 'absent' and item.public_key is defined and item.public_key != ''
+
+- hosts: web
```

Comment:
I think it would be a good idea to seperate the migration steps or at least tag them

Reply:
The next task will be some cleanup, therefor all the migration-tasks will be removed after that.